test/server/fixtures/plans/basic.json

```diff
@@ -43,6 +46,30 @@
     "usage_percentage": 10,
     "usage_type": "percentage",
     "price": {}
+  }, {
+    "add_on_type": "fixed",
+    "code": "tiered",
+    "name": "tiered",
+    "quantity": 1,
+    "usage_type": null,
+    "usage_percentage": null,
+    "optional": true,
+    "display_quantity": true,
+    "tiers": [{
+      "ending_quantity": 3,
+      "currency_code": "USD",
+      "unit_amount_in_cents": 200
+    }, {
+      "ending_quantity": 999999999,
+      "currency_code": "USD",
+      "unit_amount_in_cents": 400
+    }],
+    "tier_type": "tiered",
+    "price": {
+      "USD": {
+        "unit_amount": 4.0
+      }
+    }
   }],
   "tax_exempt": false
 }
```

Comment:
Happy to make a new fixture for this, but the tier_type should be here because it's being checked against, and will be included in the response from recurly-app from here on out.

Reply:
Great call ✅